### DISCUSSION TITLE
Websockets in Nuxt 3

### DISCUSSION THREAD

**Discussion by timb-103** [nuxt/nuxt#22149] [General]
What's the best way to setup a simple websocket server with Nuxt 3? Using a library like ws would be ideal, but can't find any good documentation or discussions recently about it.

If anyone has any ideas that would be great.

**Comment by topd5** [comment]
In my project, websockets work as part of a separate NodeJS backend . Nuxt separately, API separately, and all this is wrapped in nginx via proxy_pass.
